soc/xtensa/intel_adsp/ace/Kconfig.defconfig.ace15_mtpm

```diff
@@ -11,4 +11,8 @@ config SOC
 config MP_MAX_NUM_CPUS
 	default 3
 
+config CRYPTO
+	default y
+	depends on DT_HAS_INTEL_ADSP_SHA_ENABLED
```

Comment:
the `depends on should be in Kconfig.intel`.  We should probably set CRYPTO in the board defconfig.

Reply:
it is an implicit dependency, this file is under an `if CRYPTO`.